include/exec/task.hpp

```diff
@@ -80,13 +128,40 @@ namespace exec {
     };
 
     template <class _Ty>
-    using default_task_context = __default_task_context_impl;
+    using default_task_context = __default_task_context_impl<__scheduler_affinity::__sticky>;
+
+    template <class _Ty>
+    using __raw_task_context = __default_task_context_impl<__scheduler_affinity::__none>;
+
+    struct complete_inline_t {
+      __any_scheduler __scheduler_;
+
+      static constexpr bool await_ready() noexcept {
+        return false;
```

Comment:
Good catch, changed it to return `std::true_type`. But I think its never called anyway, since we `await_transform` that into a  `schedule(__scheduler_)`